### PR TITLE
Hotfix-FixTLS1.3

### PR DESCRIPTION
**Pull request by JonathanPitre**

### CODE REVIEW

**Comment by JonathanPitre** [comment]
I noticed some errors after more thorough testing. I fixed both adml and adjusted the strings accordingly.

File: template/de-DE/schannel.adml [hunk 1]
```diff
@@ -418,13 +426,15 @@ TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384_P521
 
 <!-- </string>If this setting is left unconfigured, TLS 1.1 and TLS 1.2 will be enabled by default for applications targeting .NET Framework 4.6 or higher and disabled otherwise. -->
 Wenn diese Einstellung unkonfiguriert bleibt, werden TLS 1.1 und TLS 1.2 standardmäßig für Anwendungen aktiviert, die .NET Framwork 4.6 oder höher verwenden. 
+https://docs.microsoft.com/en-us/dotnet/framework/network-programming/tls
```
Comment (Crosse) [inline template/de-DE/schannel.adml]: Let's use the German translation for this page.
```suggestion
https://docs.microsoft.com/de-DE/dotnet/framework/network-programming/tls
```

File: template/de-DE/schannel.adml [hunk 2]
```diff
@@ -82,6 +82,14 @@ Damit die Änderungen übernommen werden, muss der Computer neugestartet werden.
 Damit die Änderungen übernommen werden, muss der Computer neugestartet werden.
             </string>
 
+            <!-- TLSv1.3 -->
+            <string id="TLSv13">TLS 1.3 [EXPERIMENTAL]</string>
+            <string id="TLSv13_Help">Aktiviert oder deaktiviert TLS 1.3.  TLS 1.3 hat keine bekannten Sicherheitsprobleme.
+
+This setting is only compatible on Windows 10 1903 and above and does not require a reboot to take effect.
```
Comment (Crosse) [inline template/de-DE/schannel.adml]: My German is too rusty to come up with the translation on my own, but as far as I can tell (and remember), Google Translate did a pretty good job of it here:

```suggestion
Diese Einstellung ist nur unter Windows 10 1903 und höher kompatibel und erfordert keinen Neustart.
```

If a native speaker comes along (@teddue, maybe? 😄 He wrote the original translation!) and decides that this is a terrible translation, I have no qualms about changing it at that time.

**Review by Crosse — CHANGES_REQUESTED** [review]
Overall this looks good! Thank you so much. I have only a few suggested changes regarding the German translation. If those look good, go ahead and commit the suggestions and I'll merge the PR.

**Comment by JonathanPitre** [comment]
@Crosse I have completed the changes, please review thanks

**Review by Crosse — APPROVED** [review]
Perfect, I'll take it!

**Pull request merged**